cps/uploader.py

```diff
@@ -326,8 +327,11 @@ def generate_video_cover(tmp_file_path):
     ffmpeg_args = [
         ffmpeg_executable,
         '-i', tmp_file_path,
```

Comment:
Defensive programming reminder: Sanitize the file path to prevent command injection

Reply:
I'll use [shlex](https://docs.python.org/3/library/shlex.html) for now. Should be safe on POSIX compliant shells. (Done in f8c39f268c40977168bd8bacd5d3c4a6038dd46d)

Reply:
I added a general warning that `shlex.quote()` is not bulletproof:

https://github.com/iiab/calibre-web/pull/196/commits/3a965f82cf7b1368ad542905a5c13235c22808d8

(Though it's probably good enough for path sanitizing!)